product_harmonized_system_delivery/models/product_template.py

```diff
@@ -8,5 +8,5 @@ class ProductTemplate(models.Model):
     _inherit = "product.template"
 
     # IMPORTANT: Overwrites two Odoo standard fields
-    hs_code = fields.Char(related="hs_code_id.hs_code", store=True)
-    country_of_origin = fields.Many2one(related="origin_country_id", store=True)
+    hs_code = fields.Char(related="hs_code_id.hs_code")
+    country_of_origin = fields.Many2one(related="origin_country_id")
```

Comment:
Do you have a good reason for removing store=True from these 2 fields?

Reply:
In Odoo 18, it's true that you don't necessarily need to store a field to search for it. Odoo's ORM (Object-Relational Mapping) allows you to search on computed fields without storing them.

Reply:
yes, but it's bad for performance compared to store=True. Please keep as it is